src/app/rundown-view/components/rundown-view/rundown-view.component.html

```diff
@@ -1,8 +1,8 @@
 <ng-container *ngIf="rundown; else loadingRundown">
     <sofie-rundown-header [rundown]="rundown"></sofie-rundown-header>
     <sofie-rundown [rundown]="rundown"></sofie-rundown>
-    <sofie-draggable-shelf>
-        <sofie-producer-shelf></sofie-producer-shelf>
+    <sofie-draggable-shelf localStorageKeyForHeight="producer-shelf-height">
```

Comment:
This isn't an attribute that `sofie-draggable-shelf` can contain within itselves?

Reply:
I have changed the implementation a bit such that the attribute becomes `localStorageKey`. This way we can use the draggable shelf in multiple locations, and remember their positions individually.